### DISCUSSION TITLE
Using Older Version makes bootBuildImage Task Failed

### DISCUSSION THREAD

**Discussion by kworkbee** [paketo-buildpacks/feedback#64] [Q&A]
Hello.
There seems to be a bug related to telemetry in Datadog Trace Agent 1.3.0, so I want to use the downgraded version, but if you specify the previous version, the console is experiencing an error during the 'DETECTING' stage as follows.

```
 > Running creator
    [creator]     ===> DETECTING
    [creator]     ERROR: No buildpack groups passed detection.
    [creator]     ERROR: Please check that you are running against the correct path.
    [creator]     ERROR: failed to detect: no buildpacks participating
```

Please let me know how to deal with.
Thank you.

**Comment by dmikusa** [comment]
What specifically are you changing? What is in your pom.xml/build.gradle file?
